changelog.md

```diff
@@ -33,6 +33,7 @@
 * Automatically generate the libclang bindings using DStep
 * Add a switch `--rename-enum-members`, which enables stripping of superfluous prefixes from enum members.
 * Add a switch `--normalize-modules`, which enables changing the names of modules to the D conforming form.
+* Add support for unit-threaded for running unit tests.
```

Comment:
Please add an entry for the support for bitfields as well.

Reply:
Fixed.